src/test/java/io/debezium/connector/vitess/AbstractVitessConnectorTest.java

```diff
@@ -78,8 +78,10 @@
             + "text_col,"
             + "mediumtext_col,"
             + "longtext_col,"
+            + "blob_col,"
+            + "mediumblob_col,"
```

Comment:
nit: Vitess only supports blob https://github.com/vitessio/vitess/blob/04fd49ee3e5f1b7a6caf01717291954402fd8f0c/go/vt/proto/query/query.pb.go#L270-L272

might worth doing a round of auditing and see if we are missing any other types that are reasonable to sql types: https://github.com/vitessio/vitess/blob/04fd49ee3e5f1b7a6caf01717291954402fd8f0c/go/vt/proto/query/query.pb.go#L210-L309

Reply:
> nit: Vitess only supports blob.


These two are MySQL types, not Vitess types. Vitess supports all blob types supported by MySQL. 
In Vitess, they (TINYBLOB, BLOB, MEDIUMBLOB and LONGBLOG) are all treated as Blob type ( the `Type_BLOB` in your link). This is similar to TEXT types.

> might worth doing a round of auditing and see if we are missing any other types that are reasonable to sql types:

Besides [those documented](https://debezium.io/documentation/reference/stable/connectors/vitess.html#vitess-basic-types), EXPRESSION and TUPLE ard for internal use. So it should be good in terms of exhaustiveness.